### PR TITLE
LPS-63943

### PR DESCRIPTION
**Pull request by Ithildir**

### CODE REVIEW

**Comment by Ithildir** [comment]
ci:retest

**Comment by Ithildir** [comment]
ci:reopen

**Comment by jpince** [comment]
@Ithildir not sure if this is related to your changes, but those tests failed twice in a row...
```
java.lang.AssertionError: removePlugins
Expected: plugin1
     got: plugin1,ae_addimages
```

**Comment by Ithildir** [comment]
I don't think so... let me try again.

**Comment by Ithildir** [comment]
ci:retest

**Comment by jpince** [comment]
ci:stop

**Comment by jpince** [comment]
sorry, didn't look at other pulls - this is happening on all pulls.

@brianchandotcom this pull is good to merge

**Comment by jpince** [comment]
@Ithildir I found the cause https://github.com/liferay/liferay-portal/commit/e62495f58454ad4493955088e614f8f77feb0173

**Comment by jpince** [comment]
or maybe that was the fix. we'll try again

**Comment by jpince** [comment]
ci:retest

**Comment by jpince** [comment]
@Ithildir functional failure is a known CI issue. looks good, as expected :) Thanks!

**Comment by brianchandotcom** [comment]
@Ithildir I didn't merge in the change log commit since it had a conflict, https://github.com/brianchandotcom/liferay-portal/pull/45368/commits/b7d824e2adb4afe5c4d32fbb2f98138528f3c2f9

Can you resend that part? Thx.
